samples/KristofferStrube.Blazor.FileSystemAccess.WasmExample/Program.cs

```diff
@@ -10,7 +10,14 @@
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
-builder.Services.AddFileSystemAccessServiceInProcess();
+
+// Configure with custom script path
+builder.Services.AddFileSystemAccessServiceInProcess(options =>
+{
+    // The file at this path in this example is manually copied to wwwroot folder
+    // options.ScriptPath = $"/content/custom-path/{FileSystemAccessOptions.DefaultNamespace}.js";
```

Comment:
The sample start of the path should not contain `"/content/"` as we changed that structure.